tests/python/tests/test_error_messages.py

```diff
@@ -54,7 +54,10 @@ def test_encrypted_column_with_no_configuration():
 
                 sql = "INSERT INTO unconfigured (id, encrypted_unconfigured) VALUES (%s, %s)"
 
-                with pytest.raises(psycopg.Error, match='#encrypt-unknown-column'):
+                # This is EQL catching the error and returning it. Details are in docs/errors.md
+                # When mapping errors are enabled, (enable_mapping_errors or CS_DEVELOPMENT__ENABLE_MAPPING_ERRORS)
+                # Proxy will return an error that says "Column X in table Y has no Encrypt configuration"
+                with pytest.raises(psycopg.Error, match=r"Encrypted column missing \w+ \(\w+\) field"):
```

Comment:
Similarly to the other test above, we show EQL's error here when trying to insert, as we emit the warning and let the unencrypted record pass through to be caught by Postgres.